### ISSUE TITLE
Texture bug ICC

### ISSUE THREAD

**Issue by xarly** [mangosR2/scriptdev2#196]
There is a texture bug in ICC, codename "the shorcut"

After Deathbringer Saurfang, in the center of the Needle, if you go crossing the center, you fall and arrive to Lich King

**Comment by xarly** [comment]
Look like it triggers areatrigger_teleport 5718

**Comment by rsa** [comment]
yes. sometimes ago i'm manually disable this areatrigger and make script for this teleport. may be anyone broke script and don't make conditions in DB (areatrigger table) for this?
